crates/starknet/src/constants.rs

```diff
@@ -2,14 +2,20 @@ use std::net::{IpAddr, Ipv4Addr};
 
 use starknet_types::chain_id::ChainId;
 
-pub(crate) const CAIRO_0_ACCOUNT_CONTRACT_PATH: &str = concat!(
+pub const CAIRO_0_ACCOUNT_CONTRACT_PATH: &str = concat!(
     env!("CARGO_MANIFEST_DIR"),
     "/accounts_artifacts/OpenZeppelin/0.5.1/Account.cairo/Account.json"
 );
-
 pub const CAIRO_0_ACCOUNT_CONTRACT_HASH: &str =
     "0x4d07e40e93398ed3c76981e72dd1fd22557a78ce36c0515f679e27f0bb5bc5f";
 
+pub const CAIRO_1_ACCOUNT_CONTRACT_SIERRA_PATH: &str = concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/accounts_artifacts/OpenZeppelin/0.7.0/Account.cairo/Account.sierra.json"
```

Comment:
Why not the file name is only Account.sierra, we know that they are in json representation.

Reply:
I just copied what scarb generated

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/pull/196/commits/8b600906170b168daa372b54f99a5fefde453940